app/src/lib/app-state.ts

```diff
@@ -523,16 +520,38 @@ export interface ICommitSelection {
   readonly diff: IDiff | null
 }
 
-export interface IChangesState {
-  readonly workingDirectory: WorkingDirectoryStatus
+export enum ChangesSelectionKind {
+  WorkingDirectory,
```

Comment:
:art: string enums for better debuggability

```suggestion
  WorkingDirectory = 'WorkingDirectory',
```